packages/preview/invoice-maker/1.0.0/template/main.typ

```diff
@@ -0,0 +1,75 @@
+#import "@preview/invoice-maker:1.0.0": *
+
+#show: invoice.with(
```

Comment:
Just a remark: If the invoice isn't really intended to have a body, you don't _have_ to do a `show: invoice.with(` rule. It could also just be `invoice(`. But perhaps it's also good to have the flexibility.

Reply:
Yeah, I think it's good to provide the template like this,
so that people can just type away if they need to add additional infos.